deploy/cephfs/kubernetes/v1.13/csi-nodeplugin-rbac.yaml

```diff
@@ -46,8 +48,82 @@ metadata:
 subjects:
   - kind: ServiceAccount
     name: cephfs-csi-nodeplugin
+    # replace with non-default namespace name
     namespace: default
 roleRef:
   kind: ClusterRole
   name: cephfs-csi-nodeplugin
   apiGroup: rbac.authorization.k8s.io
+
+---
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: cephfs-csi-nodeplugin-psp
+spec:
+  allowPrivilegeEscalation: true
+  allowedCapabilities:
+    - 'SYS_ADMIN'
+  fsGroup:
+    rule: RunAsAny
+  privileged: true
+  hostNetwork: true
+  hostPID: true
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+    - 'configMap'
+    - 'emptyDir'
+    - 'projected'
+    - 'secret'
+    - 'downwardAPI'
+    - 'hostPath'
+  allowedHostPaths:
+    - pathPrefix: '/dev'
+      readOnly: false
+    - pathPrefix: '/sys'
+      readOnly: false
+    - pathPrefix: '/lib/modules'
+      readOnly: true
+    - pathPrefix: '/var/lib/kubelet/pods'
+      readOnly: false
+    - pathPrefix: '/var/lib/kubelet/plugins/cephfs.csi.ceph.com'
```

Comment:
cephfs.csi.ceph.com is  configurable https://github.com/ceph/ceph-csi/blob/master/charts/ceph-csi-rbd/values.yaml#L240 even plugin_registry, plugins kubelet directory are configurable  https://github.com/ceph/ceph-csi/blob/master/charts/ceph-csi-rbd/values.yaml#L240

please take care of this for  both cephfs and rbd

Reply:
I used variables defined in values.yaml in helm charts.

These are static manifests and I had to use volumes that were defined in Deployments and DaemonSets under deploy/ folder.